line_profiler/timers.c

```diff
@@ -48,11 +44,7 @@ hpTimer(void)
 {
         struct timeval tv;
         PY_LONG_LONG ret;
-#ifdef GETTIMEOFDAY_NO_TZ
-        gettimeofday(&tv);
```

Comment:
Should this line still exist for on Python <= 3.8? If not why?

Reply:
I'll quote from the commit message:

> In bpo-38068, cPython assumes gettimeofday exists and takes two arguments. That's a reasonable assumption that we can repeat.

That was a change made in Python 3.9, but the assumption is a general one, not a python-specific one. I think it's quite reasonable to make.

Reply:
So the existence of this was a quirk of necessary over-configuration? We could have always assumed `gettimeofday(*, *)` existed on POSIX and Windows platforms? I suppose its fine if that's the case.

Reply:
Yeah, you can. Python 3.9 does.

Reply:
It's a special case for ancient unixes, I presume.